dotcom-rendering/src/components/RelativeTime.importable.tsx

```diff
@@ -10,8 +10,19 @@ type Props = {
 };
 
 const ONE_MINUTE = 60_000;
+const getCurrentMinute = (then: number) =>
+	then + Math.floor((Date.now() - then) / ONE_MINUTE) * ONE_MINUTE;
 
-const getTime = () => Math.ceil(Date.now() / ONE_MINUTE) * ONE_MINUTE;
+const relativeTime = (then: number, now: number): string => {
+	const value = timeAgo(then, { now });
+	return isString(value)
+		? value
+		: new Date(then).toLocaleString('en-GB', {
+				day: 'numeric',
+				month: 'short',
+				year: 'numeric',
+		  });
+};
```

Comment:
A safe wrapper around `timeAgo`, which has the curious `string | false` signature…